exasol_script_languages_developer_sandbox/runtime/ansible/roles/docker/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: Install required system packages
+  apt:
+    name: "{{apt_dependencies}}"
+    state: latest
+    update_cache: true
+  become: "{{need_sudo}}"
+
+- name: Add Docker GPG apt Key
+  apt_key:
+    url: https://download.docker.com/linux/ubuntu/gpg
+    state: present
+  become: "{{need_sudo}}"
+
+- name: Add Docker Repository
+  apt_repository:
+    repo: deb [arch=amd64] https://download.docker.com/linux/ubuntu focal stable
+    state: present
+  become: "{{need_sudo}}"
+
+- name: Update apt and install docker-ce
+  apt:
+    name: docker-ce
+    state: latest
+    update_cache: true
+  become: "{{need_sudo}}"
+
+- name: adding docker users (for use without sudo)
+  user:
+    name: "{{user_name}}"
+    append: yes
+    groups: docker
+  become: "{{need_sudo}}"
```

Comment:
Is this because of docker. Maybe we should prepare the docker container with the same user and sudo settings as the AMI

Reply:
Created an issue: #13